docs/gke-installation.md

```diff
@@ -0,0 +1,127 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in NetworkPolicy only mode, in which no encapsulation is required for any kind of traffic
+(Intra Node, Inter Node, etc) and NetworkPolicies are enforced using OVS. Antrea is supported
+on both VPC-native Enable/Disable modes.
+
+## GKE Prerequisites
+
+1. Install the Google Cloud SDK (gcloud). Refer to [Google Cloud SDK installation guide](https://cloud.google.com/sdk/install)
+
+    ```bash
+    curl https://sdk.cloud.google.com | bash
+    ```
+
+2. Make sure you are authenticated to use the Google Cloud API
+
+    ```bash
+    export ADMIN_USER=user@email.com
+    gcloud auth login
+    ```
+
+3. Create a project or use an existing one
+
+    ```bash
+    export GKE_PROJECT=gke-clusters
+    gcloud projects create $GKE_PROJECT
+    ```
+
+## Creating the cluster
+
+You can use any method to create a GKE cluster (gcloud SDK, gcloud Console etc). The example
+given here is using the Google Cloud SDK.
+
+**Note:** Antrea is supported on Ubuntu Nodes only for GKE cluster. Also, it is a must to select service
+CIDR at the time of cluster deployment.
+
+1. Create a GKE cluster
+
+    ```bash
+    export GKE_ZONE="us-west1"
+    export GKE_HOST="UBUNTU"
+    export GKE_SERVICE_CIDR="10.94.0.0/16"
+    gcloud container --project $GKE_PROJECT clusters create cluster1 --image-type $GKE_HOST \
+       --zone $GKE_ZONE --enable-ip-alias --services-ipv4-cidr $GKE_SERVICE_CIDR
+    ```
+
+2. Access your cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                      STATUS   ROLES    AGE     VERSION
+    gke-cluster1-default-pool-93d7da1c-61z4   Ready    <none>   3m11s   v1.14.10-gke.17
+    gke-cluster1-default-pool-93d7da1c-rkbm   Ready    <none>   3m9s    v1.14.10-gke.17
+    ```
+
+3. Create a cluster-admin ClusterRoleBinding
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user user@email.com
+    ```
+
+    **Note:** To create clusterRoleBinding, the user must have `container.clusterRoleBindings.create` permission.
+Use this command to enable it, if the previous command fails due to permission error. Only cluster Admin can
+assign this permission.
+
+    ```bash
+    gcloud projects add-iam-policy-binding $GKE_PROJECT --member user:user@email.com --role roles/container.admin
+    ```
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy ``antrea-node-init`` DaemonSet to enable ``kubelet`` to operate in CNI mode.
+
+    ```bash
+    kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.5.0/antrea-gke-node-init.yml
```

Comment:
Again, maybe we talk about the location of the yaml files of the latest and released versions before steps.

Like:
> Antrea is deployed using the deployment yamls. The latest version of the yaml files can be found in https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/. Deploy a released version of Antrea from the [list of releases](https://github.com/vmware-tanzu/antrea/releases). Note that GKE support was added in release 0.5.0, which means you can not pick a release older than 0.5.0.

Then in step 1 and 2:
> To deploy a given release `<TAG>` (e.g. `v0.5.0`):
> kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-gke-node-init.yml

> To deploy the latest version:
> kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-gke-node-init.yml

Reply:
I prefer to keep antrea-gke-node-init.yml installation simple as is. We dont need to talk about release version for this, as we would be supporting just one version. If you prefer I can change the link to master, instead of 0.5.0.

Reply:
Yes, changing to master is at least better.

Reply:
I'm not planning to upload this to release assets, as I don't see a reason to. So the `releases/download` link will not be available. Since this file it not expected to change, +1 to Jianjun's suggestion.

Reply:
@antoninbas : do you suggest to remove the released version for antrea-gke.yml too?

Reply:
Yes I suggest just having `https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-gke-node-init.yml` in the instructions, as it's easier. If you really want a "released" link, you can use `https://github.com/vmware-tanzu/antrea/tree/<TAG>/build/yamls/antrea-gke-node-init.yml`. But I am not planning on making `https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-gke-node-init.yml` a valid link.

Reply:
This is based on the fact that this manifest does not point to any versioned Antrea resource (e.g. docker image) and is unlikely to change in ways that matter. If you think having a "release" link in the instructions is safer / more future-proof, feel free to use the link I paster above.